File: php/kasirmini.php
<?php

echo "Masukkan nama barang: ";
$nama = trim(fgets(STDIN));

echo "Masukkan harga barang: ";
$harga = floatval(trim(fgets(STDIN)));

echo "Masukkan jumlah beli: ";
$jumlah = intval(trim(fgets(STDIN)));

$total = $harga * $jumlah;

if ($total > 26500) {
    $diskon = $total * 0.06;
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

echo "\n===== STRUK PEMBELIAN ======\n";
echo "Nama Barang     : $nama\n";
echo "Harga Satuan    : $harga\n";
echo "Jumlah Beli     : $jumlah\n";
echo "Total Harga     : $total\n";
echo "Diskon          : $diskon\n";
echo "Total Bayar     : $total_bayar\n";
echo "=================================\n";
?>

File: cpp/kasirmini.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string nama;
    float harga;
    int jumlah;

    cout << "Masukkan nama barang: ";
    cin >> nama;

    cout << "Masukkan harga barang: ";
    cin >> harga;

    cout << "Masukkan jumlah beli: ";
    cin >> jumlah;

    float total = harga * jumlah;
    float diskon;

    if (total > 26500) {
        diskon = total * 0.60;
    } else {
        diskon = 0;
    }

    float total_bayar = total - diskon;

    cout << "\n===== STRUK PEMBELIAN =====\n";
    cout << "Nama Barang   : " << nama << endl;
    cout << "Harga Satuan  : " << harga << endl;
    cout << "Jumlah Beli   : " << jumlah << endl;
    cout << "Total Harga   : " << total << endl;
    cout << "Diskon        : " << diskon << endl;
    cout << "Total Bayar   : " << total_bayar << endl;
    cout << "============================\n";

    return 0;
}
